x-pack/test/dataset_quality_api_integration/tests/integrations/integration_dashboards.spec.ts

```diff
@@ -8,25 +8,14 @@
 import expect from '@kbn/expect';
 import { DatasetQualityApiClientKey } from '../../common/config';
 import { FtrProviderContext } from '../../common/ftr_provider_context';
-import { installPackage, IntegrationPackage, uninstallPackage } from './package_utils';
+import { installPackage, uninstallPackage } from './package_utils';
 
 export default function ApiTest({ getService }: FtrProviderContext) {
   const registry = getService('registry');
   const supertest = getService('supertest');
   const datasetQualityApiClient = getService('datasetQualityApiClient');
 
-  const integrationPackages: IntegrationPackage[] = [
-    {
-      // with dashboards
-      name: 'postgresql',
```

Comment:
`postgresql` is not available in the lite dockerized version of package registry.